client/src/pages/Register.js

```diff
@@ -0,0 +1,89 @@
+import React, { Component } from "react";
+import { withStyles } from "@material-ui/core/styles";
+//import { Router } from "react-router-dom";
+
+import axios from "axios";
+
+const registerPageStyle = theme => ({
+  landingContainer: {
+    margin: theme.spacing.unit * 2
+  }
+});
+
+class Register extends Component {
+  state = {
+    userName: " ",
+    userEmail: " ",
+    userPassword: " ",
+    confirmPassword: " "
+
+  };
+
+
+  handleSubmit = async (e) => {
+    e.preventDefault();
+    const userData = {
+      name: this.state.userName,
+      email: this.state.userEmail,
+      password: this.state.userPassword,
+      confirmPassword: this.state.confirmPassword
+    }
+    //make http post request to send name,email and password to server
+    const { data } = await axios.post("/api/users", userData)
+    console.log("new user info:", data);
+    //clear input fields
+    this.setState({
+      userName: " ",
+      userEmail: " ",
+      userPassword: "",
+      confirmPassword: " "
+    })
+
+    //direct user to profile page
+    this.props.history.replace("/profile")
+  }
+
+  handleChange = (e) => {
+
+    if (e.target.name === "userName") {
+      this.setState({ userName: e.target.value })
+    }
+    else if (e.target.name === "userEmail") {
+      this.setState({ userEmail: e.target.value })
+    }
+
+    else if (e.target.name === "userPassword") {
+      this.setState({ userPassword: e.target.value })
+    }
+
+    else {
+      this.setState({ confirmPassword: e.target.value })
+    }
+
+  }
+  render() {
+
+    const { classes } = this.props;
+    const { userName, userEmail, userPassword, confirmPassword } = this.state
+
+    return (
+      <div className={classes.landingContainer}>
+        <h1>Welcome to Twitter Audio</h1>
+        <h2> Sign up</h2>
+        <form onSubmit={this.handleSubmit}>
```

Comment:
You may wanna use material-ui components for this form. Checkout the signup tutorial at https://material-ui.com/getting-started/page-layout-examples/

Reply:
I just noticed that @gsmandur is working on this part, so disregard this comment